Podfile.lock

```diff
@@ -13,4 +13,4 @@ SPEC CHECKSUMS:
 
 PODFILE CHECKSUM: 7e6419db4fcb749f553c2e25830be7c869d3d825
 
-COCOAPODS: 1.8.4
+COCOAPODS: 1.9.1
```

Comment:
lmao i need to update my cocoapods, we keep flipping between 1.84 and 1.9.1 😂

Reply:
omg so this is why I always have to run an install